apps/mobile/src/libs/enums/app/direction.enum.ts

```diff
@@ -0,0 +1,8 @@
+const Direction = {
```

Comment:
Where do you use it? The naming is not so clear

Reply:
It is for Edges of the SafeArea. I think to move it to the ScreenWrapper, thought firstly that we might need it somewhere else. And rename to EdgeDirection.